runtime/src/main/java/com/redhat/ipaas/rest/Components.java

```diff
@@ -38,12 +35,17 @@
     @Inject
     private DataManager dataMgr;
 
+    @Context
+    private UriInfo uri;
+
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "List components")
     @ApiResponses(value = {@ApiResponse(code = 200, message = "Success", response = Component.class)})
+    @ApiParam(name = "sortField", value = "Sort list with the given field")
```

Comment:
Ah, sorry. Need to be removed as this doesn't work (was just a test). Swagger doesnt pick this up. 

I need to find out to add this externally as described in https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameters-definitions-object